src/main/java/no/nav/pto/veilarbportefolje/persononinfo/barnUnder18Aar/BarnUnder18AarService.java

```diff
@@ -49,25 +52,32 @@ public List<Fnr> hentBarnFnrsForForeldre(List<Fnr> fnrForeldre) {
         return result;
     }
 
-    public void lagreBarnOgForeldreansvar(Fnr foresattIdent, List<BarnUnder18Aar> barnFraPdl) {
-        List<Fnr> lagredeBarn = barnUnder18AarRepository.hentForeldreansvarForPerson(foresattIdent);
+    @Transactional
```

Comment:
Didnt we try this earlier? I am not quite certain what consequences this has. Can you explain?

Reply:
Yep, I am not sure that this works...need to investigate a bit.